modules/react/tabs/lib/TabsList.tsx

```diff
@@ -44,12 +44,12 @@ export const useTabsList = composeHooks(
 );
 
 const StyledStack = styled(Stack)<StyledType>({
+  maskImage: `linear-gradient(to right, white 95%, transparent)`,
   '::after': {
     content: '""',
     position: 'sticky',
     height: 52,
     minWidth: 30,
```

Comment:
I suppose we could remove the `::after` pseudo-element entirely since it's sole purpose was to enable the fade effect (can I get a confirmation on this @NicholasBoll?) and we're now applying the effect to the StyledStack itself.

Reply:
`::after` is used to prevent the use of non-semantic elements and to not fade the border. For example:

```html
<div role="tablist"> // stack with overflow and a border
  <div> // non-semantic element for the fade effect
```

We avoid these extra styling elements because you cannot get rid of them via the Component API of compound components and it serves to complicate "where to extra attributes/refs point to" and you no longer have access to all elements for easy styling. You're forced to use the Emotion styling API for nested styles - which have been an unofficial and undocumented API. Not a deal breaker, but we're trying to actively reduce elements that aren't tied to the semantic structure if possible. One current exception is `ListBox` requires multiple elements to render correctly which is unavoidable.

We may want to make this more complicated to support a fade effect only when there is something to be scrolled. This should reduce the likelihood that the last tab is active and faded which looks a bit silly.

Reply:
Removed ::after styling now that fade effect is on the StyledStack itself.